templates/block/block--matson-main-menu.html.twig

```diff
@@ -65,11 +65,17 @@
             {{ content }}
           {% endblock %}
         </nav>
+        {#
+          Temporary hack so that menu_block will render the main menu without
+          the seach box messing everything up.
+        #}
+        {% if plugin_id == 'system_menu_block:main' %}
```

Comment:
The problem here is that the main menu cannot be placed in a sidebar without the search box without this.